tests/beamsearch.ini

```diff
@@ -6,7 +6,7 @@ tf_manager=<tf_manager>
 output="tests/outputs/beamsearch"
 overwrite_output_dir=True
 batch_size=16
-epochs=2
+epochs=5
```

Comment:
proč?

Reply:
Myslim, ze mi slo o to, aby se behem toho training behu stihlo vytvorit vic model checkpointu pro testovani ensemblu (2 epochy na to ale mozna staci).

Reply:
aha, to je dobrý, to bych nechal